python/ray/util/accelerators/__init__.py

```diff
@@ -20,12 +22,27 @@
     "NVIDIA_TESLA_T4",
     "NVIDIA_TESLA_P4",
     "NVIDIA_TESLA_K80",
-    "NVIDIA_TESLA_A100",
     "NVIDIA_TESLA_A10G",
+    "NVIDIA_A100",
     "INTEL_MAX_1550",
     "INTEL_MAX_1100",
     "AWS_NEURON_CORE",
     "GOOGLE_TPU_V2",
     "GOOGLE_TPU_V3",
     "GOOGLE_TPU_V4",
+    # Deprecated
+    "NVIDIA_TESLA_A100",
 ]
+
+
+def __getattr__(name: str):
+    if name == "NVIDIA_TESLA_A100":
+        from ray.util.annotations import RayDeprecationWarning
+
+        warnings.warn(
+            "NVIDIA_TESLA_A100 is deprecated, use NVIDIA_A100 instead.",
+            RayDeprecationWarning,
+            stacklevel=2,
+        )
+        return NVIDIA_A100
+    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
```

Comment:
hmm how  does it work when other name is given in this case? (iiuc it should raise an exception?)

Reply:
It won't